caesars-palace/blocks/tabs/tabs.css

```diff
@@ -0,0 +1,253 @@
+.tabs.block .tab-navbar {
+  display: flex;
+  flex-direction: row;
+}
+
+.tabs.block .tab-title {
+  text-transform: uppercase;
+  margin-top: 8px;
+  font-size: 12px;
+  letter-spacing: 1px;
+  line-height: 0.8;
+  font-weight: 600;
+  padding-bottom: 16px;
+  text-align: center;
+}
+
+.tabs.block .hidden-tab-title {
+  display: none;
+}
+
+.tabs.block .active-tab-title {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  font-weight: 700;
+  line-height: 1.1;
+  font-size: 150%;
+  padding-top: 8px;
+  animation-name: fade-in;
+  animation-duration: 0.5s;
+}
+
+.tabs.block .backward-tab-button {
+  background: url('../../icons/chevron-left.svg') no-repeat;
+  min-width: 44px;
+  transform: scale(0.8);
+}
+
+.tabs.block .backward-tab-button:hover {
+  box-shadow: 0 2px 4px 0 rgb(0 0 0 / 10%);
+}
+
+.tabs.block .forward-tab-button {
+  background: url('../../icons/chevron-right.svg') no-repeat;
+  min-width: 44px;
+  transform: scale(0.8);
+}
+
+.tabs.block .forward-tab-button:hover {
+  box-shadow: 0 2px 4px 0 rgb(0 0 0 / 10%);
+}
+
+.tabs.block .tab {
+  box-shadow: 0 2px 4px 0 rgb(0 0 0 / 10%);
+  display: flex;
+  flex-direction: column;
+}
+
+.tabs.block .hidden-tab {
+  opacity: 0;
+
+  /* mimic display: none; but with animation support */
+  position: absolute; 
+  visibility: hidden;
+}
+
+.tabs.block .tab-image {
+  display: flex;
+}
+
+.tabs.block .tab-image img {
+  object-fit: cover;
+}
+
+.tabs.block .tab-text {
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+  padding-top: 16px;
+  padding-bottom: 16px;
```

Comment:
These can now use `var(--layout-size-m)`. You could also use padding shorthand here.

eg.
```
padding: var(--layout-size-m) 0 var(--layout-size-m) var(--layout-size-m);
```